csrc/ir/utils.h

```diff
@@ -60,13 +60,20 @@ MmaOpDetails getMmaOpDetails(
 
 void verifyMmaOpForEvaluation(MmaOp* mma_op, DataType expected_input_dtype);
 
-bool matchMatmulCast(const UnaryOp* cast_op, Val*& mma_lhs, Val*& mma_rhs);
+struct MatmulInputs {
```

Comment:
Is LinearLayerInputs a better name? MatmulInputs gives me the wrong impression that it's just a and b.